site/com_joomgallery/tmpl/gallery/default.php

```diff
@@ -72,6 +76,10 @@
     </div>
   <?php endif; ?>
 
+  <div class="gallery-header">
+    <?php echo HTMLHelper::_('content.prepare', $gallery_description); ?>
```

Comment:
I would add the gallery description where it belongs, to the GalleryModel::getItem() method. And then add it here in the template the normal way with
`<?php echo $this->item->description; ?>`

Reply:
It's changed now. Is that what you intended?